<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QWidget" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="Alumno" name="widget_8" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="Alumno" name="widget_6" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="Alumno" name="widget_5" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="Alumno" name="widget_9" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="Alumno" name="widget_3" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="Alumno" name="widget_7" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="Alumno" name="widget_4" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="Alumno" name="widget" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="Alumno" name="widget_2" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Alumno</class>
   <extends>QWidget</extends>
   <header>alumno.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
